src/runtime/GlobalObjectBuiltinReflect.cpp

```diff
@@ -25,6 +25,7 @@
 #include "runtime/NativeFunctionObject.h"
 #include "interpreter/ByteCodeInterpreter.h"
 #include "ArrayObject.h"
+#include "VMInstance.h"
```

Comment:
This header file seems unnecessary.
Other looks great :)

Reply:
VmInstance is needed to get `globalobject->toStringtag` since one of the test-case namely 
`built-ins/Reflect/Symbol.toStringTag` is testing that reflect object has his own toStringTag property.

Reply:
oh I missed that.